server/security.js

```diff
@@ -6,7 +6,10 @@ let csurf = require("csurf");
 const ONE_YEAR = 31536000000;
 
 let defaultCSPDirectives = {
-  defaultSrc: [ "'self'" ],
+  defaultSrc: [
```

Comment:
You don't want to add it here. You want to add it here: https://github.com/mozilla/thimble.mozilla.org/blob/master/server/index.js#L82-L87